web_src/css/repo.css

```diff
@@ -390,7 +390,7 @@
 }
 
 .repository.file.list .non-diff-file-content .header .file-actions .btn-octicon {
-  line-height: 1;
+  line-height: var(--line-height-default);
   padding: 10px 8px;
```

Comment:
De we need to adjust this to something like `9px 8px` to maintain previous element height?

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/26553/commits/9ba7096ba456dbf9910ee6477a3ca018308d57a8.